NetworkPolicy/k8s-listener.py

```diff
@@ -75,9 +89,9 @@ def make_rules(addr_scheme, policy_def):
 
     """
     rules = {}
-    tenant         = policy_def['owner_tenant_id']
+    tenant = policy_def['owner_tenant_id']
```

Comment:
Hrm. That's why I don't like overzealous format checking tools. The aligned stuff was much more readable. I'm inclined to allow certain exceptions (you can set those with pep8 command line options). This would be one of those.